modules/services/random-background.nix

```
@@ -41,6 +47,8 @@ in
           as a duration understood by systemd.
         '';
       };
+
+      noXinerama = mkEnableOption "Disable Xinerama support (stretch image across monitors)";
```

This will produce an odd description string, something like "Whether to enable Disable Xinerama support …".

Also, I try to avoid "negated" options, I would instead suggest calling it `enableXinerama` with the default being true. Something like
```suggestion
      enableXinerama = mkOption {
        type = types.bool;
        default = true;
        description = ''
          Will place a separate image per screen when enabled,
          otherwise a single image will be stretched across all screens.
        '';
    }
```